kb/2023-07-06/package_your_own_toolbox_image.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Package your own Toolbox Image
+description: How to package your own toolbox image
+slug: package_your_own_toolbox_image
+authors:
+  - name: Vicente Cheng
+    title: Senior Software Engineer
+    url: https://github.com/Vicente-Cheng
+    image_url: https://github.com/Vicente-Cheng.png
+tags: [debug, harvester, container]
+hide_table_of_contents: false
+---
+
+Harvester OS is an immutable OS, meaning you can't install any packages on it. Although we provide a way to [install packages](https://docs.harvesterhci.io/dev/troubleshooting/os#how-can-i-install-packages-why-are-some-paths-read-only), it is not recommended to install packages on Harvester OS. Installing extra packages on Harvester OS might cause the system to become unstable. 
```

Comment:
```suggestion
Harvester OS is designed as an immutable operating system, which means you cannot directly install additional packages on it. While there is a way to [install packages](https://docs.harvesterhci.io/dev/troubleshooting/os#how-can-i-install-packages-why-are-some-paths-read-only), it is strongly advised against doing so, as it may lead to system instability.
```

Reply:
Updated! Thanks for the review.